github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -266,3 +266,52 @@ periodics:
         resources:
           requests:
             memory: "29Gi"
+- name: periodic-kubevirt-mirror-uploader
+  cluster: ibm-prow-jobs
+  cron: "05 7 * * *"
+  max_concurrency: 1
+  decorate: true
+  decoration_config:
+    timeout: 1h
+    grace_period: 5m
+  extra_refs:
+  - org: fgimenez
+    repo: project-infra
+    base_ref: add-git-pr-script
+    workdir: true
+  - org: kubevirt
+    repo: kubevirt
+    base_ref: master
+  spec:
+    containers:
+      - image: "kubevirt/builder@sha256:e6a76ec71f2de87c3bc648bae87ab43a113540bf57da69f07f76bd0edfd500ab"
+        env:
+        - name: GIT_ASKPASS
+          value: "hack/git-askpass.sh"
+        - name: GOOGLE_APPLICATION_CREDENTIALS
+          value: /etc/gcs/service-account.json
+        command: ["/bin/sh"]
```

Comment:
I think that if you put `/entrypoint.sh` here (https://github.com/kubevirt/kubevirt/blob/master/hack/builder/Dockerfile#L109) you should have a working env.

Reply:
Thanks a lot for the pointer, the problem is that `/entrypoint.sh` uses `eval` and currently the generic PR script relies on `eval` too,  using `/entrypoint.sh` leads to nested evals and escaping issues. 

Instead I've pushed changes to download the bazelisk binary (no need for go in the path), I've also fixed some paths that were wrong bc of the different orgs for each repo (this should be removed once the generic PR script is merged), PTAL.